enacts/wat_bal/config-defaults.yaml

```diff
@@ -0,0 +1 @@
+../config-defaults.yaml
```

Comment:
Do we really need these symlinks? Can't you just use `CONFIG=../config-defaults.yaml:...`?